src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Updated to System.Diagnostics.DiagnosticSource RC version.
```

Comment:
can we explicitly list the version being updated to so people upgrading will find it easy?